### ISSUE TITLE
Can't add a new vizy block when in preview mode

### ISSUE THREAD

**Issue by anchovy** [verbb/vizy#164]
### Describe the bug

When I'm previewing a edit and I click on the '+' in the vizy field to add a new block nothing happens

### Steps to reproduce

1. Edit & preview a page with a vizy block
2. Click on the '+' to add a new block
3. Nothing happens

### Craft CMS version

Craft Pro 3.7.46

### Plugin version

1.0.16

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.16",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.17](https://github.com/verbb/vizy/releases/tag/1.0.17)
